sponsors.json

```diff
@@ -0,0 +1,17 @@
+[
+  {
+    "name": "ZapHosting",
+    "images": [
+      {"media": "(min-width: 577px)", "src": "/assets/img/sponsors/zaphosting-large.png"},
+      {"media": "(max-width: 576px)", "src": "/assets/img/sponsors/zaphosting-small.jpg"}
+    ],
+    "link": "https://zap-hosting.com/sponge"
+  },
+  {
+    "name": "BisectHosting",
+    "images": [
+      {"src": "/assets/img/sponsors/bisecthosting.svg"}
```

Comment:
These links don't work in production, because downloads is deployed at `www.spongepowered.org/downloads/`, i.e. this will point to the assets of SpongeHome. I'm not sure why it works in development environment.